game-core/src/main/java/tools/image/PolygonGrabber.java

```diff
@@ -50,14 +50,14 @@
 import tools.util.ToolLogger;
 
 /**
- * old comments
  * Utility to break a map into polygons.
- * Not pretty, meant only for one time use.
  * Inputs - a map with 1 pixel wide borders
  * - a list of centers - this is used to guess the territory name and to verify the
  * - territory name entered
  * Outputs - a list of polygons for each country
+ * @deprecated Not pretty, meant only for one time use.
  */
+@Deprecated
```

Comment:
I'm putting on my mind-reading cap here, but I suspect it was experimental code that eventually made its way into the map making tools.  The "meant only for one time use" comment may have just been carried along for the ride.

Since this class is actively reachable via the Map Making Tools command, I'm not sure we should deprecate it.  All the map making tools are "not pretty" and would probably deserve similar deprecation. :smile:

Reply:
Yeah, that would seem right; this is a pretty major class : )

Let's update the comment instead.

Reply:
ce735eb